addons/web/static/src/js/core/dom.js

```diff
@@ -539,28 +539,32 @@ var dom = {
     /**
      * @param {HTMLElement} el - the element to stroll to (limitation: if the
      *      element is using a fixed position, this function cannot work except
-     *      if is the header (with the "top" id) or the footer (with the
-     *      "bottom" id) for which exceptions have been made)
+     *      if is the header (el is then a string set to '#top') or the footer
+     *      (el is then a string set to '#bottom') for which exceptions have
+     *      been made)
      * @param {number} [options] - same as animate of jQuery
      * @param {number} [options.extraOffset=0]
      *      extra offset to add on top of the automatic one (the automatic one
      *      being computed based on fixed header sizes)
      * @param {number} [options.forcedOffset]
      *      offset used instead of the automatic one (extraOffset will be
      *      ignored too)
+     * @param {boolean} [options.isTopOrBottom]
+     *      if set, the target of the scroll is the top or the bottom of the
+     *      document depending on the value of el (either '#top' or '#bottom')
      * @return {Promise}
      */
     scrollTo(el, options = {}) {
         const $el = $(el);
-        const $scrollable = $el.parent().closestScrollable();
+        const $scrollable = options.isTopOrBottom ? $().getScrollingElement() : $el.parent().closestScrollable();
         const $topLevelScrollable = $().getScrollingElement();
         const isTopScroll = $scrollable.is($topLevelScrollable);
 
         function _computeScrollTop() {
-            if (el.id === 'top') {
+            if (el === '#top') {
                 return 0;
             }
-            if (el.id === 'bottom') {
+            if (el === '#bottom') {
```

Comment:
The util should still work the same if the top or the bottom element is given explicitly (without the string), so you have to keep the same check as before too.